cmd/admin-handlers.go

```diff
@@ -590,6 +706,16 @@ func (a adminAPIHandlers) SetConfigHandler(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
+	if err = config.Validate(); err != nil {
+		writeCustomErrorResponseJSON(w, ErrAdminConfigBadJSON, err.Error(), r.URL)
+		return
+	}
+
+	if err = config.TestNotificationTargets(); err != nil {
+		writeCustomErrorResponseJSON(w, ErrAdminConfigBadJSON, err.Error(), r.URL)
```

Comment:
In this scenario, the notification targets might be down. Which means that it is not really a bad config json.

Reply:
> In this scenario, the notification targets might be down. Which means that it is not really a bad config json.

I created this `ErrAdminConfigNotificationTargetsFailed,` to replice bad json config